src/client/atoms.ts

```diff
@@ -0,0 +1,26 @@
+import { Prisma } from '@prisma/client'
+import { atom } from 'jotai'
+
+// Display Management
+export const loginState = atom<'login' | 'signup'>('login')
+export const displayForm = atom<boolean>(false)
+
+// User Creation and Login
+export const inputName = atom<string>('')
+export const inputEmail = atom<string>('')
+export const inputPassword = atom<string>('')
+
+// User Data Store
+export const authToken = atom<string | null>(null)
```

Comment:
you could look into the atom utils to use an `atomWithLocalStorage` - cant remember the name exactly; so that this atom is automatically synced with local storage